### PR TITLE
update user manual for several post-processors

### PR DESCRIPTION
**Pull request by wangcj05**
--------
Pull Request Description
--------
Including basic statistics, metric, cross validation, importance rank, external, and data mining post processors. 
##### What are the significant changes in functionality due to this change request?

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [ ] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [ ] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [ ] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [ ] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [ ] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: doc/user_manual/DataMining.tex [hunk 1]
```diff
@@ -49,6 +49,8 @@ \subsubsection{DataMining}
   ``dr'' and the user specifies 3 components, then the output dataObject will
   have three new outputs named ``drDimension1,'' ``drDimension2,'' and
   ``drDimension3.'').
+  \nb The ``Labels'' are automatically added to the output \textbf{DataObjects}. It
+  is also accessible by the users using the variable name defined in above.
```
Comment (PaulTalbot-INL) [inline doc/user_manual/DataMining.tex]: Remove "in" from "defined in above".

File: doc/user_manual/postprocessor.tex [hunk 2]
```diff
@@ -114,9 +114,15 @@ \subsubsection{BasicStatistics}
...
     \item \textbf{samples}: the number of samples in the data set used to determine the statistics.
   \end{itemize}
-  If all the quantities need to be computed, this can be done through the \xmlNode{all} node, which
-  requires the \xmlNode{targets} and \xmlNode{features} sub-nodes.
-
+  This XML node needs to contain the attribute:
+  \begin{itemize}
+    \itemsep0em
+    \item \xmlAttr{prefix}, \xmlDesc{required string attribute}, user-defined prefix for the given \textbf{metric}.
+      For scalar quantifies, RAVEN will define a variable with name defined as:  ``prefix'' + ``\_'' + ``parameter anme''.
```
Comment (PaulTalbot-INL) [inline doc/user_manual/postprocessor.tex]: spelling "anme".

In addition, a "for example" would clarify things a lot for this section, one for each scalar and matrix.  I know there's examples below, but just one demonstration would help, such as (e.g. mean_x1) and (e.g. covar_x1_x2).

File: doc/user_manual/postprocessor.tex [hunk 3]
```diff
@@ -114,9 +114,15 @@ \subsubsection{BasicStatistics}
...
+      For matrix quantities, RAVEN will define a variable with name defined as: ``prefix'' + ``\_'' + ``target parameter name'' + ``\_'' +``feature parameter name''. 
+      This variable will be used by RAVEN for the internally calculations. It is also accessible by the user through
```
Comment (PaulTalbot-INL) [inline doc/user_manual/postprocessor.tex]: "for the internally calculations": either "for internal calculations" or "internally" is fine

File: doc/user_manual/postprocessor.tex [hunk 4]
```diff
@@ -1702,10 +1679,12 @@ \subsubsection{ETImporter}
 
 \subsubsection{Metric}
 \label{MetricPP}
-The \textbf{Metric} post-processor is specifically used to calculate the distance values among points and histories,
+The \textbf{Metric} post-processor is specifically used to calculate the distance values among points from PointSet and histories from HistorySet,
```
Comment (PaulTalbot-INL) [inline doc/user_manual/postprocessor.tex]: PointSet and HistorySet should be plural, if I understand your meaning.  If not, then it should be "among points from a PointSet" and etc.

File: doc/user_manual/DataMining.tex (hunk 1 above)
Comment (wangcj05) [reply doc/user_manual/DataMining.tex]: fixed.

File: doc/user_manual/postprocessor.tex (hunk 2 above)
Comment (wangcj05) [reply doc/user_manual/postprocessor.tex]: fixed

File: doc/user_manual/postprocessor.tex (hunk 3 above)
Comment (wangcj05) [reply doc/user_manual/postprocessor.tex]: fixed

File: doc/user_manual/postprocessor.tex (hunk 4 above)
Comment (wangcj05) [reply doc/user_manual/postprocessor.tex]: fixed.

**Comment by PaulTalbot-INL** [comment]
No code changes, so good to go.

**Pull request merged**